frontend/src/pages/TermPlanner/main.less

```diff
@@ -1,23 +1,23 @@
 html,
 body {
   height: 100%;
+  --option-header-height: 65px;
 }
 
 .content {
   @small: ~"(max-width: 1400px)";
 
   .mainContainer {
-    height: calc(100vh - 70px);
+    height: calc(100vh - var(--navbar-height) - var(--option-header-height));
+    padding-top: 0.5em;
     overflow: auto;
   }
 
   .plannerContainer {
     display: flex;
     // align-items: center;
-    margin-top: 0.5em;
+    margin: 1em 1em 1em 1em;
```

Comment:
Just a note, for units, might be better of to go with rem instead of em? rem is sizing relative to the root while em is relative to the parent node. So if the parent size changes, this could be changed aswell which may not be intend. Personally, I always go with rem when I need it as it keep things tied relative to the root node.

Also, for margin and padding, I usually stick with using px instead so that may be a better option? :)

Though if either rem/em messes up the stylings, don't worry bout it!

Reply:
ohh yup makes sense!